server/services/AI/AI.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs');
+
+/* Random card picked*/
+exports.autoPickCard = cards => {
+    const randomCard = cards[Math.floor(Math.random()*cards.length)];
+    return randomCard.cardId;
+}
+
+exports.autoWordHello = () => {
+    return "hello";
+}
+
+// Generate list of labels for each picture //
+async function autoWordGenerator() {
+    // Imports the Google Cloud client library
+    const vision = require('@google-cloud/vision');
+  
+    // Creates a client
+    const client = new vision.ImageAnnotatorClient();
+  
+    // Performs label detection on the image file
+    var cardId;
+    for (cardId=1; cardId < 248; cardId++) {
+        const [result] = await client.labelDetection('./src/images/cards/card (' + cardId + ').jpg');
+        const labels = result.labelAnnotations;
+        let descriptions = [];
+        labels.forEach(label => descriptions.push(label.description));
```

Comment:
This can be rewritten as a map e.g. `const descriptions = labels.map(label => label.description)`